client/src/App.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+<<<<<<< HEAD
+=======
+  
+>>>>>>> parent of 08b33aa... Revert "Updated css variables with another font that was needed. Also cleaned up App and removed default React setup code."
     </div>
```

Comment:
Can you fix this piece

Reply:
I accidentally edited in the wrong branch but I should have reverted back. Hopefully all should be good now.